networkx/drawing/nx_pylab.py

```diff
@@ -373,7 +374,10 @@ def draw_networkx_nodes(
         Line width of symbol border
 
     edgecolors : [None | scalar | sequence] (default = node_color)
-        Colors of node borders
+        Colors of node borders. Can be a single color or a sequence of colors with the
+        same length as nodelist. Color can be string or rgb (or rgba) tuple of floats
+        from 0-1. If numeric values are specified they will be mapped to colors
+        using the cmap and vmin,vmax parameters. See matplotlib.scatter for more details.
```

Comment:
We can use intersphinx, to make this even better (I think the following should work, but need to test)
```suggestion
        using the cmap and vmin,vmax parameters. See `~matplotlib.pyplot.scatter` for more details.
```